### PR TITLE
Block ip v3 pb

### PR DESCRIPTION
**Pull request by ssokolovich**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: #41271 ([link](https://github.com/demisto/etc/issues/41271))

## Description
Added a new playbook for Block IP V3. This is a new version for the previous one (v2) that includes:
- More integrations
- Fixed inputs for sub-playbooks
- User interaction for blocking process
- Deleted deprecated commands/sub-playbooks

## Screenshots
Block_IP_-_Generic_v3

## Minimum version of Cortex XSOAR
- [X] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_IP_-_Generic_v3_README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+This playbook blocks malicious IPs using all integrations that are enabled. The direction of the traffic that will be blocked is determined by the XSOAR user ( and set by default to outgoing)
```
[the remark this answers is not in the document]
Comment (ssokolovich) [reply Packs/CommonPlaybooks/Playbooks/playbook-Block_IP_-_Generic_v3_README.md:1]: Changed: "The direction of the traffic that will be blocked is determined by the XSOAR user ( and set by default to outgoing)"
To - _"By default, the blocked communication will be set to the outgoing (The blocked IP will be set as the destination)."_
WDYT?

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_IP_-_Generic_v3.yml [hunk 2]
```diff
@@ -0,0 +1,3675 @@
...
+    Specify the Static Address Group name for IPs list handling.
+  playbookInputQuery:
+- key: IPListName
+  value: {}
+  required: false
+  description: |-
+    This input determines whether Palo Alto Networks Panorama or Firewall External Dynamic Lists are used for blocking IPs.
+    Specify the EDL name for IPs list handling.
+  playbookInputQuery:
+- key: EDLServerIP
```
[the remark this answers is not in the document]
Comment (ssokolovich) [reply Packs/CommonPlaybooks/Playbooks/playbook-Block_IP_-_Generic_v3.yml:3519]: That was something I copied from the V2 version. Just updated the playbook with the previous playbook without changes for the input.

**Comment by ShirleyDenkberg** [comment]
@David-BMS Doc review completed.

**Comment by ssokolovich** [comment]
Hey @ShirleyDenkberg,
I did some additional changes in the playbook inputs and some other tasks. Any chance you can review the delta?
Many thanks!

File: Packs/CiscoFirepower/Playbooks/Cisco_FirePower-_Append_network_group_object.yml [hunk 3]
```diff
@@ -0,0 +1,464 @@
...
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
```
Comment (David-BMS) [inline Packs/CiscoFirepower/Playbooks/Cisco_FirePower-_Append_network_group_object.yml:386]: Can be removed

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_IP_-_Generic_v3.yml [hunk 4]
```diff
@@ -0,0 +1,3569 @@
...
+  description: |-
+    This input determines whether Palo Alto Networks Panorama or Firewall Dynamic Address Groups are used.
+    Specify the Dynamic Address Group tag name for IPs list handling.
+  playbookInputQuery:
+- key: ResolveIP
+  value:
+    simple: "False"
+  required: true
+  description: |-
+    This input is required for the IP Enrichment - Generic v2. Will be used for resolving the IP hostname. Default: False.
```
Comment (David-BMS) [inline Packs/CommonPlaybooks/Playbooks/playbook-Block_IP_-_Generic_v3.yml:3466]: If you removed the subplaybook then change the description to match

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_IP_-_Generic_v3.yml [hunk 5]
```diff
@@ -0,0 +1,3569 @@
+id: Block IP - Generic v3
+version: -1
+name: Block IP - Generic v3
+description: "This playbook blocks malicious IP addresses using all integrations that\
```
Comment (David-BMS) [inline Packs/CommonPlaybooks/Playbooks/playbook-Block_IP_-_Generic_v3.yml:4]: Notice how to describe the PAN-OS various options and that the user may not need to enable all of them

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_IP_-_Generic_v3_README.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+* akamai-add-elements-to-network-list
+* aria-block-dest-subnet
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| IP | An array of malicious IPs to block. Enter a comma-separated list of IPs \(i.e: 1.1.1.1,2.2.2.2\). |  | Optional |
+| CustomBlockRule | This input determines whether Palo Alto Networks Panorama or Firewall Custom Block Rules is used.<br/>Specify True to use Custom Block Rules. <br/>If set to true, this option will create 2 FW rules inside the PAN-OS device. | True | Optional |
```
Comment (David-BMS) [inline Packs/CommonPlaybooks/Playbooks/playbook-Block_IP_-_Generic_v3_README.md:77]: Notice the confusing false option

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_IP_-_Generic_v3.yml [hunk 7]
```diff
@@ -0,0 +1,3675 @@
...
+    }
+  }
+inputs:
+- key: IP
+  value: {}
+  required: false
+  description: 'An array of malicious IPs to block. Enter a list of IPs separated
+    by comma delimiter (i.e: 1.1.1.1,2.2.2.2).'
+  playbookInputQuery:
+- key: CustomBlockRule
```
[the remark this answers is not in the document]
Comment (David-BMS) [reply Packs/CommonPlaybooks/Playbooks/playbook-Block_IP_-_Generic_v3.yml:3418]: Notice confusing false option

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_IP_-_Generic_v3.yml [hunk 8]
```diff
@@ -0,0 +1,3569 @@
...
+  required: false
+  description: |-
+    This input determines whether Palo Alto Networks Panorama or Firewall Static Address Groups are used.
+    Specify the Static Address Group name for IPs list handling.
+  playbookInputQuery:
+- key: Tag
+  value: {}
+  required: false
+  description: |-
+    This input determines whether the External Dynamic Lists option is used for blocking IPs.
```
Comment (David-BMS) [inline Packs/CommonPlaybooks/Playbooks/playbook-Block_IP_-_Generic_v3.yml:3451]: Start the sentence with 
This input provides the tag name with which indicators will get tagged with.

**Review by David-BMS — APPROVED** [review]
Please finalize the comments before merging

**Pull request merged**